esi_report.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# run ./esi_report.sh daily to get daily report
+# run ./esi_report.sh monthly to get last month's report
+# run ./esi_report.sh weekly to get last week's report
+# run ./esi_reports.sh plan to get future plan
+
+report_file=/tmp/esi_report
+
+source /home/stack/overcloud-deploy/overcloud/overcloudrc
+
+write_output () {
+    openstack esi lease list -f value |
+    while read line
+    do
+        start_time=$(cut -d' ' -f5 <<<$line)
+        end_time=$(cut -d' ' -f6 <<<$line)
+        if [[ $end_time < $first_day || $start_time > $last_day ]]; then
+            :
+        else
+            echo $line >> $output_file
+        fi
```

Comment:
I think it's possible to pass in a time range so that this isn't needed.

Reply:
Sure! I thought --time-range <start_time, end_time> strictly lists leases that start&end in this specific time range. I just realized that it's not the case.

Reply:
It may be worth putting in a PR to fix the tooltip in python-esileapclient, as what the time range does isn't quiiiite what the comment says.